docs/user-manual/en/amqp-broker-connections.md

```diff
@@ -32,121 +32,145 @@ To define an AMQP broker connection, add an `<amqp-connection>` element within t
+These messages are then transferred to brokerB:5672. A link with the address name $ACTIVEMQ_ARTEMIS_MIRROR_brokerB will be created towards brokerB. If there is a security manager configured, security roles must be provided to the user on the broker connection.
+
+Notice the queue $ACTIVEMQ_ARTEMIS_MIRROR_brokerB will not physically exist on brokerB and there won't be a way to find it on the administrator console. The broker will treat these messages accordingly as mirror events and will execute the appropriate operations at the target broker.
+
+### Pre Existing Messages
+The brokers will only send messages from the point in time the mirror was configured. Previously existing messages will not be forwarded to other brokers.
+
+## Dual Mirror (Disaster and Recovery option)
+
+ActiveMQ Artemis supports automatic failback mirroring. Every sent message and every acknowledgement is asynchronously replicated to the mirrored broker.
+
+On the following diagram, there will be two servers called DataCenter1, and DataCenter2. In order to have a dual mirror configuration, it is necessary is to add the mirror broker connection on each broker.xml:
+
+![Broker Connection DR](images/broker-connection-DR.jpg)
+Figure 2. Broker Connection Disaster and Recovery.
 
+The broker connection will replicate sends and acknowledgements no matter where they originated. If messages are sent on DC1 (DataCenter1) these will be automatically transitioned to DC2 (DataCenter2). Messages Acknowledged on DC2 will be transitionally back to DC1. 
+The only exception to that rule would be if there were consumers with pending messages on any server. It is recommended to not have active consumers on both servers.
+
+## Example
+There is an example as part of the distribution showing dual broker configuration (or disaster and recovery) under ./examples/features/broker-connection/disaster-recovery.
+
+On the example two brokers are configured to mirror each other and whatever happens in one broker is immediately copied over to the other broker.
 
 ## Peers
-The broker can be configured as a peer which connects to the [Apache Qpid Dispatch Router](https://qpid.apache.org/components/dispatch-router/) and instructs it the broker it will act as a store-and-forward queue for a given AMQP waypoint address configured on the router. In this scenario, clients connect to a router to send and receive messages using a waypointed address, and the router routes these messages to or from the queue on the broker.
+The broker can be configured as a peer which connects to the [Apache Qpid Dispatch Router](https://qpid.apache.org/components/dispatch-router/) and instructs it that the broker will act as a store-and-forward queue for a given AMQP waypoint address configured on the router. In this scenario, clients connect to a router to send and receive messages using a waypointed address, and the router routes these messages to or from the queue on the broker.
```

Comment:
I would suggest that this entire 'Peers' section be moved later after the "## Senders and Receivers" section, as its probably the less likely to be used, and it would better match up with the order of their outline in the doc intro.

Reply:
good idea.. will do